packages/containers/src/actionAPI.js

```diff
@@ -16,14 +19,13 @@ function evalExpressions(action, context, payload = {}) {
 		context,
 		payload,
 	);
-	if (action.labelExpression) {
-		delete newAction.labelExpression;
-		newAction.label = api.expression.call(action.labelExpression, context, newAction);
-	}
-	if (action.iconExpression) {
-		delete newAction.iconExpression;
-		newAction.icon = api.expression.call(action.iconExpression, context, newAction);
-	}
+	forIn(action, (value, key) => {
+		if (key.match(regexExpression)) {
+			const newKey = key.slice(0, -10);
```

Comment:
Guess you could use a regexp with capture to avoid slicing based on `Expression` length.

Reply:
its more safe to